src/stores/Binding/Store.ts

```diff
@@ -765,6 +798,39 @@ const getInitialState = (
         );
     },
 
+    setCollectionMetadata: (values, addedCollections) => {
+        if (!hasLength(values)) {
+            return;
+        }
```

Comment:
I think we need a way to handle this to be used to clear out the settings when all the backfills are removed.

Was this here just to be safe or are we relying on this for something?

Reply:
No, this is very important and remember there are multiple triggers for the alert at hand and often the action is called over a subset of collections.